docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -397,6 +397,18 @@ The following command operators are supported:
 
   Renders a numeric value in typed JSON logs.
 
+%DOWNSTREAM_HANDSHAKE_DURATION%
+  HTTP
+    Not implemented ("-").
```

Comment:
This could be treated as a generic transport socket field, as it is now, and be tcp-only. But if you moved it into the `sslConnection()` and made it TLS-specific, it would work for http requests as well, without any additional work to copy it from connection to request StreamInfo.  (See `StreamInfoSslConnectionInfoFieldExtractor` in substitution_formatter.cc). I'd be inclined to move it to TLS-specific, unless you think someone is going to implement this on other transport sockets. WDYT? Also @alyssawilk any opinion here?

Reply:
That works, my only argument for leaving as-is is that it's nicely grouped and discoverable with other timing metrics in downstreamTiming. Another option is to thread it through to copy to streaminfo (shouldn't be too tricky if we think it's worth it). I'm pretty new to Envoy so defer to you and Alyssa on this one.

Reply:
@ggreenway @alyssawilk can you give @pksohn some guidance on this so we can move this forward? Thanks!

Reply:
May be a question for @ianswett .  I believe that Google will want timing metrics for LOAS eventually, and while that plug-in is in house and I don't know if we'll upstream the similar ALTS implementation, AFIK it makes sense to keep it generic.

Reply:
Yes, we'd want it for LOAS/ALTS.  I'm not sure if it's relevant, but we also want it for QUIC and HTTP/3.

Reply:
@ggreenway seems like the consensus so far is to leave the implementation as-is, OK with you?